src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/spec/proceed/confirmation/JudgmentByAdmissionConfHeader.java

```diff
@@ -14,7 +14,8 @@ public class JudgmentByAdmissionConfHeader implements RespondToResponseConfirmat
 
     @Override
     public Optional<String> generateTextFor(CaseData caseData) {
-        if (CaseState.All_FINAL_ORDERS_ISSUED == caseData.getCcdState()) {
+        if (CaseState.All_FINAL_ORDERS_ISSUED == caseData.getCcdState()
```

Comment:
For being extra catutious, Do we need to use JO flag here for not impacting PROD?